src/converters/piral-ng/src/startup.ts

```diff
@@ -11,11 +21,19 @@ function getVersionHandler(versions: Record<string, () => void>) {
   return versions[version];
 }
 
+// Equivalent to platformBrowserDynamic, but with support for multiple platforms
+const customPlatformDynamicFactory = createPlatformFactory(platformCoreDynamic, 'piralDynamic', [
+  ...INTERNAL_BROWSER_DYNAMIC_PLATFORM_PROVIDERS,
+  {
+    provide: ALLOW_MULTIPLE_PLATFORMS,
+    useValue: true,
+  },
+]);
 const runningModules: Array<[any, NgModuleInt, PlatformRef]> = [];
 
 function startNew(BootstrapModule: any, context: ComponentContext, ngOptions?: NgOptions) {
   const path = context.publicPath || '/';
-  const platform = platformBrowserDynamic([
+  const platform = customPlatformDynamicFactory([
```

Comment:
This is awesome - the only question I have is regarding in what version(s) of Angular this is available?

We'd accept it anyway, but if this is not available, e.g., in Angular 6 or lower then we'd need to document this (and make clear that `piral-ng` only works for Angular 7+).

Reply:
That's a good point.

From looking at the git blame all the methods and tokens used stem from mid 2016 (https://github.com/angular/angular/commit/979657989bab4fb93faf6a207d30760faa9bd696) and the 4.0.0-betas in February 2017 (https://github.com/angular/angular/commit/17486fd696a49fd2d11fe295ec5add8bb44fb398).

So I think this should be fine :)